src/rtc-to-net/rtc-to-net.ts

```diff
@@ -175,7 +165,7 @@ module RtcToNet {
       });
       var writeSnapshot = () => {
         this.getSnapshot().then((snapshot:RtcToNetSnapshot) => {
-          log.info('snapshot: %1', [JSON.stringify(snapshot)]);
+          log.info('snapshot: ' + JSON.stringify(snapshot));
```

Comment:
Two reasons:
 - First, logging ain't working right, so I want to simplify what we're doing with logging.  Hopefully it'll eventually help
 - Let's just construct and keep the string, and not retain the entire JS object.